src/main/java/org/jenkinsci/plugins/android_lint/LintPublisher.java

```diff
@@ -96,23 +112,32 @@ public String getPattern() {
         return pattern;
     }
 
+    /**
+     * Sets the Ant file-set pattern of files to work with.
+     */
+    @DataBoundSetter
+    public void setPattern(final String pattern) {
+        this.pattern = pattern;
+    }
```

Comment:
It is preferable to use `@DataBoundSetter` on a, well, setter :-) rather than on a field, for the usual reasons: better encapsulation, and makes future API changes easier to do compatibly.